modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Select/MultipleSelect.tsx

```diff
@@ -74,8 +74,7 @@ export function MultipleSelect({
 	useEffect(() => {
 		if (selectAllOption) {
 			let firstRender = false;
-
-			let notAllSelected: MultiSelectItemChild | undefined;
+			let allSelected = true;
 
 			(options as MultiSelectItem[]).forEach(({children}) => {
```

Comment:
Hey @igor-franca, we already have the type of this `option`, so we can remove this type declaration here